packages/editor/src/browser/editor-manager.ts

```diff
@@ -47,6 +48,9 @@ export class EditorManager extends NavigatableWidgetOpenHandler<EditorWidget> {
      */
     readonly onCurrentEditorChanged: Event<EditorWidget | undefined> = this.onCurrentEditorChangedEmitter.event;
 
+    @inject(ApplicationShell)
+    protected readonly applicationShell: ApplicationShell;
```

Comment:
please revert this file, `EditorManager` already has apis to get all editors, try `all`

Reply:
Thanks for clarification Anton! I did not see this API and got confused for a while.